### ISSUE TITLE
Unable to click/hover a text element with an offset

### ISSUE THREAD

**Issue by flp-gpc** [mapbox/mapbox-gl-js#6146]
Hi,

I have a simple text element with a click event:
```
{
  "id": "mytext",
  "type": "symbol",
  "layout": {
    "text-field": "{name}\n{elevation} m",
    "text-font": ["DIN Offc Pro Bold"],
    "text-max-width": 20,
    "text-size": 15
  },
  "paint": {
    "text-color": "#fff",
    "text-halo-width": 1.3,
    "text-halo-color": "rgba(0,0,0,0.6)"
  }
}
```
```
this.map.on('click', 'mytext', e => {
  new mapboxgl.Popup()
	.setLngLat(e.features[0].geometry.coordinates)
	.setHTML('hello')
	.addTo(this.map);
  });
}
```
Everything is working fine, when I click on my text the popup is displayed.

However, if I add an offset:
```
"text-offset": [0,2],
```
It is not possible to click on the text anymore. The same issue with mouseenter or mouseleave, it seems the event is not triggered anymore. Also the same issue with icons.

**Comment by anandthakker** [comment]
Duplicate of https://github.com/mapbox/mapbox-gl-js/issues/6075

**Comment by phiphou** [comment]
I dont understand ? This issue is fixed ? Will be ? (sorry for the flood, not a github expert)

**Comment by anandthakker** [comment]
The issue is fixed in the `master` branch, and will be available in the next release (0.45)

**Comment by phiphou** [comment]
Ok, I now understand well. Thanks for all ;)
